tests/RobotFramework/requirements/requirements.txt

```diff
@@ -2,7 +2,7 @@ dateparser~=1.1.7
 paho-mqtt~=1.6.1
 python-dotenv~=0.20.0
 robotframework~=6.0.0
-robotframework-c8y @ git+https://github.com/reubenmiller/robotframework-c8y.git@0.13.4
+robotframework-c8y @ git+https://github.com/reubenmiller/robotframework-c8y.git@0.14.2
```

Comment:
Why is it pointing to your repo here?

Reply:
Both the `robotframework-c8y` and `robotframework-devicelibrary` are currently hosted on my owner at the moment. I will be transferring ownership of these libraries eventually to either `SoftwareAG` or `ThinEdge` organisations once I have finished some housekeeping tasks (e.g. publishing to pip, automated test runs, improved documentation).